### ISSUE TITLE
can't delete members from groups

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#1536]
looks like an SKS bug:

```
Error: protection.remove_cryptgroup_user: Error in function (type, msg, asError = TRUE)  : Empty reply from server
R trace:
fun(structure(list(message = msg, call = sys.call()), class = c(typeName, 
    "GenericCurlError", "error", "condition")))
function (type, msg, asError = TRUE) 
{
    if (!is.character(type)) {
        i = match(type, CURLcodeValues)
        typeName = if (is.na(i)) 
            character()
        else names(CURLcodeValues)[i]
    }
    typeName = gsub("^CURLE_", "", typeName)
    fun = (if (asError) 
        stop
    else warning)
    fun(structure(list(message = msg, call = sys.call()), class = c(typeName, 
        "GenericCurlError", "error", "condition")))
}(52L, "Empty reply from server", TRUE)
.Call("R_curl_easy_perform", curl, .opts, isProtected, .encoding, 
    PACKAGE = "RCurl")
curlPerform(curl = curl, .opts = opts, .encoding = .encoding)
RCurl::getURL(paste0(getConf("session.server"), request))
.session.server.request(paste0("/mod_group?token=", URIencode(token), 
    "&realm=", URIencode(realm), "&group=", URIencode(group), 
    action))
strsplit(.session.server.request(paste0("/mod_group?token=", 
    URIencode(token), "&realm=", URIencode(realm), "&group=", 
    URIencode(group), action)), "\n", TRUE)
session.server.modify.group("rcloud", .session$token, groupid, 
    remove = user)
fun(...)
```

**Comment by s-u** [comment]
I'm trying to trace this, but the GUI doesn't allow me to add any members to the group - if I tab, enter or press ok, the text you type in simply disappears. What is the prerequisite to add a member?

**Comment by anatoliyg** [comment]
You cannot type in a member, you'd need to select one from the auto complete drop down and press OK. Is that drop down appearing for you?

**Comment by anatoliyg** [comment]
like so.
to get this working locally I had to sign into rcloud with a couple
different github id's so they get added to the users list

[image: Inline image 2]

**Comment by s-u** [comment]
Got it - the problem was that the user list seems to be cached so if you add a user in the meantime it doesn't get updated. I was able to add it after re-loading the session.

**Comment by s-u** [comment]
Indeed a SKS bug, should be fixed by s-u/SessionKeyServer@90add8b
